README.md

```diff
@@ -180,26 +185,26 @@ curl --location --request POST 'http://localhost:9195/shenyu/plugin/selectorAndR
   
   ![](https://shenyu.apache.org/img/shenyu/dataSync/config-strategy-processor-en.png)
 
---------------------------------------------------------------------------------    
+---    
 
 # Prerequisite
  
    * JDK 1.8+
    
---------------------------------------------------------------------------------    
-    
-# Document & Website
-
-[![EN doc](https://img.shields.io/badge/document-English-blue.svg)](https://shenyu.apache.org/docs/index)
-[![CN doc](https://img.shields.io/badge/document-Chinese-blue.svg)](https://shenyu.apache.org/zh/docs/index/)
-  
---------------------------------------------------------------------------------  
+--- 
         
 # Stargazers over time
 
 [![Stargazers over time](https://starchart.cc/apache/incubator-shenyu.svg)](https://starchart.cc/apache/incubator-shenyu.svg)
 
---------------------------------------------------------------------------------  
+---  
+
+# Discussion and Support
+
+* [Mailing Lists](mailto:dev@shenyu.apache.org)
+* QQ Group: 429951241
```

Comment:
remove qq group

Reply:
removed

Reply:
not commit？

Reply:
Emma, yeah, I don't notice it failed to push.